src/client/webpack.config.js

```diff
@@ -43,7 +42,12 @@ const webpackConfig = {
     ]),
     new webpack.optimize.CommonsChunkPlugin(/* chunkName= */'vendor', /* filename= */'vendor.js', function(module){
       return module.resource && module.resource.indexOf('node_modules') !== -1;
-    })
+    }),
+    new webpack.DefinePlugin({
+     __CONFIG__: args.endpoint
```

Comment:
This is a good demo app, and for people unfamiliar with webpack and only reading the bootstrapper.js, it looks like magic that it works and not great for discovering how it all works together.  cc @KeithWoods